build.sbt

```diff
@@ -6,16 +6,12 @@ version := "0.0.1"
 scalaVersion := "2.12.12"
 
 libraryDependencies += "org.apache.spark" %% "spark-sql" % "3.3.1" % "provided"
-libraryDependencies += "org.apache.spark" %% "spark-hive" % "3.3.1" % "provided"
 
 libraryDependencies += "io.delta" %% "delta-core" % "2.1.0" % "provided"
 libraryDependencies += "com.github.mrpowers" %% "spark-daria" % "0.38.2"
```

Comment:
Is this dependency needed / should it be made a test dependency?

Reply:
We can leave be as a test dependency. I will do the change.